tests/figures/chart/gauge_rendering.test.ts

```diff
@@ -53,11 +52,11 @@ describe("Gauge rendering config", () => {
 
   test("Chart title", () => {
     expect(getRenderingConfig(testRuntime).title).toEqual({
-      label: testRuntime.title,
+      label: testRuntime.title.content,
       fontSize: GAUGE_TITLE_FONT_SIZE,
       textPosition: {
         x: GAUGE_TITLE_PADDING_LEFT,
-        y: GAUGE_TITLE_PADDING_TOP + GAUGE_TITLE_FONT_SIZE,
+        y: NaN, //TODO : Is it possible to test this ?
```

Comment:
NaN ?

Reply:
Yes, it seems like we don't have any real size here in the test, so the computation of the position (base on the dimensions) give a nan